draft-ietf-tls-tls13.md

```diff
@@ -1530,8 +1563,8 @@ in its original KeyShare.
 
 Upon re-sending the ClientHello and receiving the
 server's ServerHello/KeyShare, the client MUST verify that
-the selected CipherSuite and NamedGroup match that supplied in
-the HelloRetryRequest. If either of these values differ, the client
+the selected NamedGroup matches that supplied in
+the HelloRetryRequest. If these values differ, the client
 MUST abort the connection with a fatal "handshake_failure" alert.
```

Comment:
nit: doesn't need a separate sentence anymore; can just be:
"
Upon re-sending the ClientHello and receiving the
server's ServerHello/KeyShare, the client MUST verify that
the selected NamedGroup matches the one supplied in
the HelloRetryRequest, and MUST abort the connection
with a fatal "handshake_failure" alert if it does not.
"